ext/hal/st/lib/sensor/vl53l0x/api/platform/src/vl53l0x_platform.c

```diff
@@ -1,275 +0,0 @@
-/*******************************************************************************

```

Comment:
lets not remove the example code

Reply:
Personally, I prefer to remove it in order not to have 2 files that can be confusing for the user. If user want the original file, it is in git history or on ST website.